app/Member_project.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Member_project extends Model
```

Comment:
Pls replace Member_project to "MemberProject"

Reply:
yep sir